a-seed-from-nothing.sh

```diff
@@ -144,21 +139,24 @@ kolla_neutron_ml2_tenant_network_types:
 EOF
 fi
 
+# Set Environment variables for Kayobe dev scripts
+export KAYOBE_CONFIG_SOURCE_PATH=~/deployment/src/kayobe-config
+export KAYOBE_VENV_PATH=~/deployment/venvs/kayobe
+
```

Comment:
Can these be exported by sourcing the beokay env-vars script?

Reply:
They're specific to the Kayobe dev scripts unfortunately, so they're not set by the env-vars script and it would be a bit unnecessary to add them to it